src/Store.java

```diff
@@ -0,0 +1,168 @@
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+public class Store {
+
+    static Bill oneUah = new Bill(1, 0);
+    static Bill twoUah = new Bill(2, 0);
+    static Bill fiveUah = new Bill(5, 0);
+    static Bill tenUah = new Bill(10, 2);
+    static Bill twentyUah = new Bill(20, 0);
+    static Bill hundredUah = new Bill(100, 10);
+    static Bill[] Bills = new Bill[]{oneUah, twoUah, fiveUah, tenUah, twentyUah, hundredUah};
+
+    static ArrayList<String> operationsLog = new ArrayList<String>();
```

Comment:
Prefer to declare a variable as an interface, rather than implementation. In this particular case, `List<String> operationsLog = new ArrayList<>()`